src/sentry/templates/sentry/organization-members.html

```diff
@@ -82,7 +82,7 @@
       </thead>
       <tbody>
         {% for member, needs_sso, has_2fa in member_list %}
-          <tr>
+          <tr {% if member.role == highlight_role %}class="member-role-highlight"{% endif %}>
```

Comment:
Can we just simply do:

```
{% if member.role == request.GET.role %}
```

And avoid the python view? I forget if `request` is always available or not. Then this can be isolated entirely to the template code without needing to modify the view itself.